pkg/kubefedctl/enable/enable.go

```diff
@@ -282,6 +282,36 @@ func CreateResources(cmdOut io.Writer, config *rest.Config, resources *typeResou
 	} else if err != nil {
 		return errors.Wrapf(err, "Error getting CRD %q", resources.CRD.Name)
 	} else {
+		ftcs := &fedv1b1.FederatedTypeConfigList{}
+		err := client.List(context.TODO(), ftcs, namespace)
+		if err != nil {
+			return errors.Wrap(err, "Error getting FederatedTypeConfig list")
+		}
+
+		for _, obj := range ftcs.Items {
+			ftc := &fedv1b1.FederatedTypeConfig{}
+			err = client.Get(context.TODO(), ftc, namespace, obj.Name)
+			if err != nil {
+				return errors.Wrapf(err, "Error getting FederatedTypeConfig %q", obj.Name)
+			}
+
+			if existingTypeConfig.Name == ftc.Name {
+				continue
+			}
+
+			var name string
+			fedType := ftc.Spec.FederatedType
+			if len(fedType.Group) == 0 {
+				name = fedType.PluralName
+			} else {
+				name = fmt.Sprintf("%s.%s", fedType.PluralName, fedType.Group)
+			}
+
+			if name == existingCRD.Name {
+				return errors.Errorf("Federated type CRD %q is used by FederatedTypeConfig %q. Try to specify an another group by --federation-group.", name, ftc.Name)
```

Comment:
-> Failed to enable federation of %q(`concreteTypeConfig.Name`) due to the FederatedTypeConfig for %q(`ftc.Name`) already referencing a federated type CRD named %q(`name`). If these target types are distinct despite sharing the same kind, specifying a non-default --federation-group should allow %q(`concreteTypeConfig.Name`) to be enabled.